src/FormErrors.php

```diff
@@ -56,11 +56,10 @@ public function getErrorSummaryFirstErrors(): array
 
     public function getFirstError(string $attribute): string
     {
-        if (empty($this->attributesErrors[$attribute])) {
-            return '';
-        }
-
-        return reset($this->attributesErrors[$attribute]);
+        return match (empty($this->attributesErrors[$attribute])) {
```

Comment:
I don't think it is necessary to use match here. It was alright.

Reply:
`Match()` does a strict match plus we only get one return, which is nice.